### PR TITLE
Update the update tool learn page

### PR DESCRIPTION
**Pull request by keizer619**

### CODE REVIEW

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ This section introduces various terms used throughout this guide. We recommend t
 
 ### Ballerina tool
 
-**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build and run programs, etc. 
+**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build and run programs, etc.
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:41]:
```suggestion
**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
```

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ This section introduces various terms used throughout this guide. We recommend t
...
 
 It also enables you to easily install, update and switch among Ballerina distributions. The main focus of this guide is to teach you how, but first, let’s talk about Ballerina distributions.
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:43]:
```suggestion
It also enables you to easily install, update, and switch among Ballerina distributions. The main focus of this guide is to teach you how to perform these actions but first, let’s talk about Ballerina distributions.
```

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 3]
```diff
@@ -56,26 +56,26 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
 
 ### Release channels
 
-Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. We don’t yet have a release channel for nightly builds that give you access to the latest, perhaps unstable features. 
+Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. We don’t yet have a release channel for nightly builds that give you access to the latest, perhaps unstable features.
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:59]:
```suggestion
Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
```

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 4]
```diff
@@ -56,26 +56,26 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
...
 
-Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers. 
+Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers.
 
 #### Patch release channel
 
-This channel gives you access to patch releases of Ballerina distributions that contain bug fixes and fixes for critical stability and security issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well. 
+This channel gives you access to patch releases of Ballerina distributions that contain bug fixes and fixes for critical stability and security issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:65]:
```suggestion
This channel gives you access to the patch releases of Ballerina distributions that contain bug fixes and fixes for critical stability and security related issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
```

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 5]
```diff
@@ -56,26 +56,26 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
...
+This channel gives you access to feature releases of Ballerina distributions. Ballerina programs that you’ve written today should continue to work on these minor releases. There will be four minor releases a year. You will get access to a minor release on the 3rd Wednesday of the 3rd month of every quarter.
 
-*Example minor releases: jballerina 1.1.0, jballerina 1.2.0, jballerina 1.3.0*
+*Example minor releases: 1.1.0, 1.2.0, 1.3.0*
 
 #### Release maintenance
 
 - We maintain a minor release 1.x.0 by issuing a series of patch releases 1.x.y. The maintenance of a particular minor release stops when there are two newer minor releases available.
-- In other words, patch releases for jBallerina 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when jBallerina 1.2 is available, we stop maintaining jBallerina 1.0.0.
+- In other words, patch releases for 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when 1.2 is available, we stop maintaining 1.0.0.
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:78]:
```suggestion
- In other words, patch releases for 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when 1.2 is available, maintaining 1.0.0 will be stopped.
```

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 6]
```diff
@@ -151,14 +151,14 @@ Most of these subcommands are self-explanatory. Therefore, the following section
 
 The `ballerina dist update` command updates your active distribution to the latest patch version.
 
-E.g., If the active distribution in your environment is “jballerina-1.1.0” and there exists patch version “jballerina-1.1.4” in our servers, this command will fetch and set it as the active distribution.
+E.g., If the active distribution in your environment is 1.1.0” and there exists patch version 1.1.4” in our servers, this command will fetch and set it as the active distribution.
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:154]:
```suggestion
E.g., If the active distribution in your environment is 1.1.0 and there exists the patch version 1.1.4 in our servers, this command will fetch and set it as the active distribution.
```

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 7]
```diff
@@ -151,14 +151,14 @@ Most of these subcommands are self-explanatory. Therefore, the following section
...
 ```sh
 → sudo ballerina dist update
-Fetching the latest patch distribution for 'jballerina-1.0.4' from the remote server...
-Fetching the 'jballerina-1.0.5' distribution from the remote server...
-Downloading jballerina-1.0.5 100% [==========================================================] 96/96 MB
-Successfully set the latest patch distribution 'jballerina-1.0.5' as the active distribution
+Fetching the latest patch distribution for '1.0.4' from the remote server...
+Fetching the '1.0.5' distribution from the remote server...
+Downloading 1.0.5 100% [==========================================================] 96/96 MB
+Successfully set the latest patch distribution '1.0.5' as the active distribution
```
Comment (praneesha) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:161]:
```suggestion
Successfully set the latest patch distribution '1.0.5' as the active distribution.
```

Comment (keizer619) [reply swan-lake/learn/keeping-ballerina-up-to-date.md:161]: This is an command output

Comment (praneesha) [reply swan-lake/learn/keeping-ballerina-up-to-date.md:161]: +1 no need to update then.
